firebase-installations/src/main/java/com/google/firebase/installations/time/Clock.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.installations.time;
+
+/**
+ * Provides the current value of "now" to allow injecting time as a dependency for testing.
+ *
+ * @hide
+ */
+public interface Clock {
+  /** Returns the current time in milliseconds since EPOCH in UTC. */
+  long now();
```

Comment:
Please call this #millis() in accordance with the interface you are trying to simulate:
https://developer.android.com/reference/java/time/Clock#millis()